### DISCUSSION TITLE
B.3.2. Ontologies & Mapping - EKG/Maturity

### DISCUSSION THREAD

**Discussion by giscus** [EKGF/ekg-maturity#94] [Page Comments]
# B.3.2. Ontologies & Mapping - EKG/Maturity

Maturity Model for the Enterprise Knowledge Graph (EKG)

https://maturity.ekgf.org/pillar/organization/delivery-management/ontologies-and-mapping/

**Comment by rivettp** [comment]
In EKGF we have not been distinguishing :"taxonomies" as something distinct from ontologies or vocabularies.

**Comment by jgeluk** [comment]
Ah ok, will fix that.
